Tools/i18n/pygettext.py

```diff
@@ -613,7 +683,9 @@ class Options:
     make_escapes(not options.escape)
 
     # calculate all keywords
-    options.keywords.extend(default_keywords)
+    options.keywords = {kw: {0: 'msgid'} for kw in options.keywords}
```

Comment:
`--keyword` still works but the keywords are assumed to be single-argument. A followup PR could add support for more.